lib/chart/internal/axis_line.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Chart.Internal.AxisLine do
+  @moduledoc false
+
+  alias Chart.Internal.AxisLine.Helpers
+
+  defguard scale(s) when s in [:linear, :log]
+
+  def new() do
+    %{
+      scale: :linear,
+      thickness: 2,
+      line: nil
+    }
+  end
+
+  def add(settings, key) when is_map(settings) and is_atom(key) do
+    settings
+    |> put_in([:plot, :axis], [key | settings.plot.axis])
```

Comment:
Tomuto nerozumim? Co to dela?

Reply:
List `:axis` obsahuje vsechny definovane osy. Je to  z toho duvodu, ze kdyz se treba zavola `Plot.set_size({600, 500})` tak aby se zmenily `MajorTicks...` pozice apod.List `:axis` obsahuje vsechny definovane osy. Je to  z toho duvodu, ze kdyz se treba zavola `Plot.set_size({600, 500})` tak aby se zmenily `MajorTicks...` pozice apod.